rockcraft.yaml

```diff
@@ -0,0 +1,131 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+# https://github.com/apache/superset/blob/master/Dockerfile
+name: charmed-superset-rock
+base: ubuntu@22.04
+version: 2.1.0-22.04-edge
+summary: Charmed Superset ROCK OCI
+description: |
+  Superset is fast, lightweight, intuitive, and loaded with options
```

Comment:
Would it be better if this description started with something like `Superset is a business intelligence tool that is fast, ...`. Because this description does not tell what Superset really is if you do not know it already.

Reply:
Good point, I've updated this.